src/Identifier/Ldap/AdapterInterface.php

```diff
@@ -35,7 +35,7 @@ public function bind(string $bind, string $password): bool;
      * @param array $options Additional options
      * @return void
      */
-    public function connect(string $host, int $port, array $options): void;
+    public function connect(string $host, int $port, array $options, bool $tls): void;
```

Comment:
couldnt tls default to false to make this less hard a bc break?

Reply:
It does default to false, see default config on the src/Identifier/LdapIdentifier.php

Reply:
the default port number is set in the default config too